distros/spring/src/test/java/com/connexta/transformation/service/TransformControllerTest.java

```diff
@@ -47,13 +50,13 @@
 
   @Autowired private MockMvc mockMvc;
 
-  @MockBean TransformationManager tManager;
-
-  @MockBean(answer = Answers.RETURNS_MOCKS)
-  private RestTemplateBuilder restTemplateBuilder;
+  @MockBean InMemoryTransformationManager manager;
 
   @Test
-  public void testAcceptedResponseEntity() throws Exception {
+  public void testCreatedResponseEntity() throws Exception {
```

Comment:
❗️ We should we add a test where we actually set the proxy headers to verify that the location URL is properly created.

Reply:
☝️ Last test required here.